plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-paypal-proxy-controller.php

```diff
@@ -234,9 +252,9 @@ private function get_paypal_access_token() {
 			'timeout'   => 45, // TODO: Set a timeout.
 			'sslverify' => false, // TODO: Set sslverify.
 		);
-		error_log( '(Proxy) PayPal access token request: ' . wc_print_r( $args, true ) );
+		error_log( '(Proxy) PayPal access token request: ' . wc_print_r( $this->sanitize_args( $args ), true ) );
 
-		$response  = wp_remote_post( 'https://api-m.sandbox.paypal.com/v1/oauth2/token', $args );
+		$response  = wp_remote_post( $this->get_paypal_access_token_request_url( $testmode ), $args );
```

Comment:
Not a blocker but a different suggestion for the main proxy. We can create only one function to get the main URL and append the specific path inline if we want to avoid creating a new function to get each API endpoint.

```suggestion
		$response  = wp_remote_post( $this->get_paypal_request_url( $testmode ) . '/v1/oauth2/token', $args );
```